src/pages/CreateClassroomPage.jsx

```diff
@@ -348,6 +356,8 @@ function CreateClassroomPage(props) {
                                         id="name"
                                         className="form-control bg-light-pastel-blue fs-5 border-0 rounded-4 mb-3"
                                         onChange={(e) => setClassroom({ ...classroom, name: e.target.value })}
+                                        minLength="3"
+                                        required
```

Comment:
```suggestion
                                        minLength="3"
                                        maxLength="20"
                                        required
```
- Nome também possui limite superior na API;